packages/web-sdk/src/instrumentations/session/instrumentation.ts

```diff
@@ -77,7 +62,10 @@ export class SessionInstrumentation extends BaseInstrumentation {
 
     if (isUserSessionValid(userSession)) {
       sessionId = userSession?.sessionId;
-      sessionAttributes = userSession?.sessionMeta?.attributes;
+      sessionAttributes = {
+        ...(userSession?.sessionMeta?.attributes ?? {}),
```

Comment:
Fallbacking to `{}` is not needed in the case of `undefined` or `null`

Reply:
Oh reall, nice. TIL!